### PR TITLE
Updated class Sample

### PR DESCRIPTION
**Pull request by DavidMansolino**
Update of the class Sample used for the controller:
-Add of the three BackLeds.
-Add of the Gyroscope.

### CODE REVIEW

File: projects/robots/darwin-op/controllers/sample/Sample.hpp [hunk 1]
```diff
@@ -38,8 +39,12 @@
       Servo                           *mServos[NSERVOS];
       LED                             *mEyeLED;
       LED                             *mHeadLED;
+      LED                             *mBackLedRed;
+      LED                             *mBackLedGreen;
+      LED                             *mHeadLedBlue;
       Camera                          *mCamera;
       Accelerometer                   *mAccelerometer;
+      Gyro							  *mGyro;
```
Comment (fabienrohrer) [inline projects/robots/darwin-op/controllers/sample/Sample.hpp:47]: Small indentation issue.

Comment (fabienrohrer) [inline projects/robots/darwin-op/controllers/sample/Sample.hpp:47]: Just a comment about our way to work. (It's not important at all, I just try to increase my skills in github)

I can't change that indentation issue simply from the Github GUI. Normally the logical procedure would be to reject this pull until this is fixed (in another pull request). (This is surely overkilled for this small issue, I accept is of course, I just try to find the best way to work with this tool).
I guess we can resolve these kind of issues by creating branches. Then the branch IS the pull request, and you can modify it after having created the pull request, in the same thread of discussion.
=> I will document myself on that

Comment (DavidMansolino) [inline projects/robots/darwin-op/controllers/sample/Sample.hpp:47]: Ok, keep me informed when you know more about that.

Comment (fabienrohrer) [inline projects/robots/darwin-op/controllers/sample/Sample.hpp:47]: Ok, in this case, if you continue your commits, they are concatenated to this pull request.
Working with branches is fundamental (amongs other) to solve this kind of issue.
I expect that "one pull request = one branch" is a good rule.
I begin to appreciate a lot github ^^

**Pull request merged**